docs/mathematics-algorithms/fermat's-Little-Theorem.md

```diff
@@ -0,0 +1,174 @@
+---
+id: fermat's-Little-theorem
+title: Fermat's Little Theorem
+sidebar_label: Fermat's Little Theorem MMI
+sidebar_position: 5
+description: A comprehensive guide to calculating the modular multiplicative inverse using Fermat's Little Theorem.
```

Comment:
replace `description: A comprehensive guide to calculating the modular multiplicative inverse using Fermat's Little Theorem.` with `description: "A comprehensive guide to calculating the modular multiplicative inverse using Fermat's Little Theorem."`